README.md

```diff
@@ -0,0 +1,61 @@
+# Money Transfer Project
+
+## Prerequisites
+
+Before running this application, ensure you have the following installed:
+
+* [.NET 8.0 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/8.0) or later
+* Temporal Server
+
+## Steps to get started
+
+## Start the Temporal Server
+Run the command: 
+`temporal server start-dev`
+
+## Clone the Repo
+Follow these steps to copy to you local machine. 
+
+```
+git clone https://github.com/jsundai/money-transfer-temporal-template-dotnet.git
+
+cd money-transfer-temporal-template-dotnet
+```
+
+## Restore Dependencies 
+Run the following command: 
+```
+dotnet restore 
+```
```

Comment:
Usually not needed as a separate step, happens on `dotnet run` by default I believe